### DISCUSSION TITLE
标签功能优化

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4938] [公告]
- Bot 现在完全支持 Discussion 和 Pull Request 了
  - `新反馈` 标签重命名为 `新提交`，以适应 Pull Request
  - 不能通过将标签改为 `完成` 来合并 Pull Request :/
- 添加了操作标签 `➦ 锁定`、`➦ 解锁`，可用于操作 Issues、Pull Requests 和 Discussions（#3535）
- 操作标签 `➦ 删除` 现可用于删除 Discussion
  - 由于 API 限制，无法用于删除 Pull Request
- 被 Reopen 的 Issue 将被自动添加 `等待确认` 标签，而非 `新反馈` 标签
- Issue 可以在被关闭时保留 `暂无计划`、`受阻碍`、`等待提交者` 标签，此前它们不能被用于已关闭的 Issue
  - 现共有 6 种标签可以作用于开启或关闭的 Issue：`暂无计划`、`受阻碍`、`第三方`、`等待提交者`、`需要社区复现`、`需要社区帮助`
- 优先级标签 `🟦 等待` 现被合并入 `暂无计划`
- 在任何条目更改后，Bot 会 **等待一分钟** 再进行自动化处理，以免在改到一半的时候和 Bot 打起来

**Comment by MoYuan-CN** [comment]
Discussion 大区貌似不会处理 `重复` 标签（操作后已经超过 1 分钟）？

**Reply by LTCatt** [reply]
是哪个 discussion……？

**Reply by MoYuan-CN** [reply]
- #5049
我过了一阵子去看没关掉我手动操作了

**Reply by LTCatt** [reply]
hmm Discussion 的标签不会被处理（因为一般也不会加标签过去……）

**Reply by MoYuan-CN** [reply]
好吧

**Comment by MoYuan-CN** [comment]
有计划为 Bot 加上投票帖子同步原帖标题的功能吗（虽然大半年不会有一次就是了）
- https://github.com/Hex-Dragon/PCL2/discussions/227#discussioncomment-10282968

**Reply by LTCatt** [reply]
这个手动改改吧……

**Comment by MoYuan-CN** [comment]
不会对 PR 修改标签
- #5021

**Comment by Deep-Dark-Forest** [comment]
@LTCatt 感觉可以让六角龙机器人在 PR 被创建后自动给这个 PR 添加 “新提交” 标签？

**Reply by LTCatt** [reply]
PR 的数量很少，感觉没啥必要……

**Reply by Deep-Dark-Forest** [reply]
> PR 的数量很少，感觉没啥必要……

额，那把 “新反馈” 改成 “新提交” 的意义也不是很大啊……
毕竟 Collaborator 看到以后要么改成等待确认要么关了，这个标签在 PR 上完全没有起到作用

**Comment by LinQingYuu** [comment]
>不能通过将标签改为 完成 来合并 Pull Request :/

~所以真的有人这样干吗?~

**Reply by Deep-Dark-Forest** [reply]
可能是龙猫自己这么做比较方便吧
